cmd/cluster/kubevirt/create.go

```diff
@@ -40,6 +41,7 @@ func NewCreateCommand(opts *core.CreateOptions) *cobra.Command {
 	cmd.Flags().StringVar(&opts.KubevirtPlatform.RootVolumeAccessModes, "root-volume-access-modes", opts.KubevirtPlatform.RootVolumeAccessModes, "The access modes of the root volume to use for machines in the NodePool (comma-delimited list)")
 	cmd.Flags().StringVar(&opts.KubevirtPlatform.ContainerDiskImage, "containerdisk", opts.KubevirtPlatform.ContainerDiskImage, "A reference to docker image with the embedded disk to be used to create the machines")
 	cmd.Flags().StringVar(&opts.KubevirtPlatform.ServicePublishingStrategy, "service-publishing-strategy", opts.KubevirtPlatform.ServicePublishingStrategy, fmt.Sprintf("Define how to expose the cluster services. Supported options: %s (Use LoadBalancer and Route to expose services), %s (Select a random node to expose service access through)", IngressServicePublishingStrategy, NodePortServicePublishingStrategy))
+	cmd.Flags().StringVar(&opts.KubevirtPlatform.CacheStrategyType, "cache-strategy-type", opts.KubevirtPlatform.CacheStrategyType, "Set the boot image caching strategy; Supported values:\n- \"None\": no caching (default).\n- \"PVC\": Cache into a PVC; only for QCOW image; ignored for container images")
```

Comment:
for the cli flag, what would you think about calling it `--root-volume-cache-strategy`. It aligns a little closer to our existing root volume flags.

the api field "CacheStrategy"  has some more context than this `--cache-strategy-type`. The CacheStrategy field is a subfield of the KubevirtRootVolume struct, so we know that the caching is related to the root volume. but for the flag, `cache-strategy-type` might be a bit too generic, since there could be other caching besides root volume images in the future.